web/docs/advanced/jobs.md

```diff
@@ -2,81 +2,137 @@
 title: Recurring Jobs
 ---
 
-import { Required } from '@site/src/components/Required.tsx'
+import { Required } from '@site/src/components/Required'
+import { ShowForTs, ShowForJs } from '@site/src/components/TsJsHelpers'
 
-If you have server tasks that you do not want to handle as part of the normal request-response cycle, Wasp allows you to make that function a `job` and it will gain some "superpowers."
+In typical web apps, users send a request to the server and receive a response containing some data. If the time to get the data is short, the app feels responsive and smooth. But what happens if the server needs to do some longer work before it can respond?
+
+For instance, sending an email or making a slow HTTP request to an external API. In such cases, you might want to execute the task in the background, without blocking the user's request.
```

Comment:
I saw the changes, but I'll keep the discussion in this thread.

I still prefer my suggestion to the current version.

I liked your initial idea of asking a rhetorical question, why not keep it? Also, the current sentence is a bit long. The object changes its number throughout the sentence (one task in the first part -> multiple tasks at the end).

Finally, if the server must perform the task _before_ responding, how can background jobs help us (that's why I changed the wording on that part too)?

One small thing (if you decide to keep the current text):
- "To avoid making the user wait" -> Too many verbs, IMO. Why not "To spare the user from waiting"